internal/docstore/firedocstore/fs.go

```diff
@@ -35,7 +35,7 @@ import (
 type collection struct {
 	client    *vkit.Client
 	dbPath    string // e.g. "projects/P/databases/(default)"
-	collPath  string // e.g. "projects/P/databases/(default)/documents/MyCollection"
+	collPath  string // e.g. "projects/P/databases/(default)/documents/States/Wisconsin/cities"
```

Comment:
Why make this change here? Does `States/Wisconsin/cities` also represent some type of hierarchy?

Reply:
I wanted to show that `collPath` could be the full path to a nested collection.